README.md

```diff
@@ -315,16 +266,260 @@ For example:
+
+Create an appropriate user mapping:
+```sql
+    	CREATE USER MAPPING
+	FOR pguser
+	SERVER influxdb_svr
+    	OPTIONS (
+	  user 'username',
+	  password 'password'
+	);
+```
+Where `pguser` is a sample user for works with foreign server (and foreign tables).
+
+### Create foreign table
+All `CREATE FOREIGN TABLE` SQL commands can be executed as a normal PostgreSQL user if there were correct `GRANT USAGE ON FOREIGN SERVER`. No need PostgreSQL supersuer for security reasons but also works with PostgreSQL supersuer.
+
+Create a foreign table referencing the InfluxDB table:
+You need to declare a column named `time` to access InfluxDB time column.
+```sql
+	CREATE FOREIGN TABLE t1(
+	  time timestamp with time zone,
+	  tag1 text,
+	  field1 integer
+	)
+	SERVER influxdb_svr
+	OPTIONS (
+	  table 'measurement1'
+	);
+```
+You can use `tags` option to specify tag keys of a foreign table.
+```sql
+	CREATE FOREIGN TABLE t2(
+	  tag1 text,
+	  field1 integer,
+	  tag2 text,
+	  field2 integer
+	)
+	SERVER influxdb_svr
+	OPTIONS (
+	  tags 'tag1, tag2'
+	);
+```
+You can import foreign schema
+```sql
+	IMPORT FOREIGN SCHEMA public
+	FROM SERVER influxdb_svr
+	 INTO public;
+```
+Access foreign table
+```sql
+	SELECT * FROM t1;
+```
+Limitations
+-----------
+
+- `UPDATE` is not supported.
+- `WITH CHECK OPTION` constraints is not supported.
+Following limitations originate from data model and query language of InfluxDB.
+- Result sets have different number of rows depending on specified target list.
+For example, `SELECT field1 FROM t1` and `SELECT field2 FROM t1` returns different number of rows if
+the number of points with field1 and field2 are different in InfluxDB database.
+- Timestamp precision may be lost because timestamp resolution of PostgreSQL is microseconds while that of InfluxDB is nanoseconds.
+- Conditions like `WHERE time + interval '1 day' < now()` do not work. Please use `WHERE time < now() - interval '1 day'`.
+
+When a query to foreign tables fails, you can find why it fails by seeing a query executed in InfluxDB with `EXPLAIN VERBOSE`.
+    
+Contributing
+------------
 Opening issues and pull requests on GitHub are welcome.
 
-## License
+Useful links
+------------
+
+### Source code
+
+Reference FDW realisation, `postgres_fdw`
+ - https://git.postgresql.org/gitweb/?p=postgresql.git;a=tree;f=contrib/postgres_fdw;hb=HEAD
+
+### General FDW Documentation
+
+ - https://www.postgresql.org/docs/current/ddl-foreign-data.html
+ - https://www.postgresql.org/docs/current/sql-createforeigndatawrapper.html
+ - https://www.postgresql.org/docs/current/sql-createforeigntable.html
+ - https://www.postgresql.org/docs/current/sql-importforeignschema.html
+ - https://www.postgresql.org/docs/current/fdwhandler.html
+ - https://www.postgresql.org/docs/current/postgres-fdw.html
+
+### Other FDWs
+
+ - https://wiki.postgresql.org/wiki/Fdw
+ - https://pgxn.org/tag/fdw/
+ 
+License
+-------
 Portions Copyright (c) 2018-2021, TOSHIBA CORPORATION
```

Comment:
can you help me edit to this format?

----

Copyright (c) 2018, TOSHIBA CORPORATION
Copyright (c) 2011-2016, EnterpriseDB Corporation

Reply:
Fixed.

Reply:
It seems it lose a line break.
Could you please check it again?

Reply:
Thanks, @weiting1lee. I also update dates.

Reply:
Thank you.